README.md

```diff
@@ -1 +1,272 @@
-# Taskcluster configuration for Servo
+# Taskcluster configuration for Servo CI
+
+This repository contains tools and configuration for Servo’s use of
+[Taskcluster](https://community-tc.services.mozilla.com/docs/) for testing pull requests.
+
+As much as possible of the configuration is kept “in tree”
+[in the `servo/servo` repository](https://github.com/servo/servo/tree/master/etc/taskcluster)
+so that changes (for example `apt install`’ing a new system dependency in a `Dockerfile`)
+can be made in the same pull request that relies on them.
+This reduces the need for coordination across multiple repositories.
+See also the `README` there for more on Servo’s testing workflow.
+
+Some other parts of the configuration are not practical to deploy automatically
+when merging a pull request, and require semi-manual steps. They are kept here.
+
+
+## Requirements
+
+Ensure that you have `python3` and `virtulenv` executables in your `$PATH`.
+The commands below will automatically create a virtual environment
+and install Python dependencies in it.
+
+
+## Obtaining administrative Taskcluster credentials
+
+Most (but not all) commands described below require running in a terminal where this has been done.
+
+```sh
+eval `mach creds`
+```
+
+This command should open a new browser tab.
+If that page says “Sign in to create a client”,
+click the “Sign in” button on the top right then “Sign in with GitHub”.
+Click on the save / floppy disk icon on the bottom right.
+
+This exports environment variables that other commands rely on.
```

Comment:
Unfortunately I can't do that :( because it's requiring me to give taskcluster access to all my organizations, and no way(?) to customize it. This is a bit of a roadblock.

Reply:
Do you mean when signing in to https://community-tc.services.mozilla.com/ with GitHub? I don’t think we (Servo) have any control over that as it’s a bug / limitation in either the "Community-TC sign-in" app (https://bugzilla.mozilla.org/enter_bug.cgi?product=Taskcluster&component=Services would be the place to file a bug) or in GitHub itself :(

Reply:
Let's discuss this in issue #13.